server/utils/azureAppInsights.ts

```diff
@@ -15,9 +14,7 @@ function version(): string {
 }
 
 export function initialiseAppInsights(): void {
-  // Loads .env file contents into | process.env
-  config()
-  if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+  if (process.env.APPLICATIONINSIGHTS_CONNECTION_STRING) {
```

Comment:
this env variable was already in our helm deploy configuration, just not being used